website/content/projects/2019/spec.md

```diff
@@ -403,9 +404,54 @@ You are welcome to create pull requests that do not affect the master branch, an
+At several points throughout the project, each student will provide feedback to two other students in their same team. This is similar to a peer evaluation/review (which are fairly common in large software projects), except the feedback you provide will have no effect on the recipient's grade (whereas in a real project, these evaluations/reviews could affect whether you get a raise, a promotion, etc.). Instead, we will be assessing the feedback you provide to others.
+
+At the end of Sprints 2, 3, and 4, you will be assigned two peers who will be providing you with feedback (you will know each other's identities). Before they provide you with feedback, you must think about what you would like to receive feedback on. At the end of Sprints 2 and 3, you should think about the following: 
+
+> *What is something I'd like to improve or do better in the next sprint?*
+
+At the end of Sprint 4, you should think about the following:
+
+> *What is something I'd like to improve or do better the next time I work in a complex software project and/or the next time I work in a team?*
+
+Based on this, you should prepare prompts for specific feedback, which can range from organizational/teamwork aspects (e.g., wanting to improve how you communicate with the rest of the team) to technical aspects (e.g., wanting to improve your understanding of some programming concepts), as well as from concrete to high-level. You are also welcome to provide additional prompts that don't originate from the above questions. Once you have your prompt(s), you should convey them to the students who will be providing you with feedback (you can communicate this directly to them, via Slack, e-mail, etc.; you do not need to include the instructor or TAs in this communication)
+
+For example, you could provide prompts like the following (the first two are examples of relatively high-level prompts, while the latter three are more specific):
+
+1. I feel like I'm using Slack and GitHub effectively, but I wonder if there are any ways in which I could be using them more effectively.
+2. I always procrastinate on my tasks until the 1-2 days before they are due, and I'd like to make more steady progress throughout the sprint. Do you have any suggestions on how I could do this?
+3. Sometimes I feel like I go off and work on one of my tasks, and when I touch base with everyone else after a few days, some of the work I've done turns out to be headed in the wrong direction, and it seems like that could've been avoided if I had known what everyone else was doing. How could I make sure I stay informed of everyone else's progress?
+4. I feel like I sometimes derail conversations by getting too lost in the weeds when talking about my work, even though I may have a valuable point to make. How could I avoid doing this?
```

Comment:
I might even offer feedback providers the opportunity to disagree here. "Do you also feel like I'm doing this? If so, what other ways do you think I could make this point instead of the way I'm doing it now?"

It might be that the feedback giver _doesn't_ feel that the conversations are getting derailed...which is useful for the recipient to know!

Reply:
I really like this, I'll include it in Example 4!